e2e/testdata/fn-eval/image-pull-policy-always/.expected/setup.sh

```diff
@@ -0,0 +1,6 @@
+#! /bin/bash
+
+# Intentionally tag a wrong image as gcr.io/kpt-fn/set-labels:v0.1, since we
+# expect the correct image to be pulled and override the wrong image.
+docker pull gcr.io/kpt-fn/starlark:v0.1
+docker tag gcr.io/kpt-fn/starlark:v0.1 gcr.io/kpt-fn/set-labels:v0.1
```

Comment:
you'r messing with a global state here. Are you making any assumptions about the order of functions in order for this to not interfere with other function that are run after? Also don't assume that the function will clean up after itself (because it can fail at any time).

Do you want to use a function that's not used by other functions to isolate the side effects?

Reply:
Updated